telemetry/specs/INT.mdk

```diff
@@ -1483,6 +1677,22 @@ INT Metadata Header and Metadata Stack, followed by UDP payload:
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
 
+
```

Comment:
IMO it would be cleaner to add a section header. Change the title of the previous example, replacing "INT" with "INT-MD", then add a title here with "INT-MX", referring to the previous section for the IP and UDP headers. Start with the INT shim header with type = 3 and length = 3 here.

Reply:
Done.